packages/core/src/rum/instrumentation/DdRumResourceTracking.tsx

```diff
@@ -131,7 +131,7 @@ export function calculateResponseSize(xhr: XMLHttpRequest): number {
         break;
     }
   } catch (e) {
-    console.error("Couldn't get resource size, because of the error", e);
+    InternalLog.log("Couldn't get resource size, because of the error", SdkVerbosity.ERROR);
```

Comment:
Because of which error? 
```suggestion
    InternalLog.log("Couldn't get resource size, because an error occurred: " + e, SdkVerbosity.ERROR);
```

Reply:
Yup, my bad, I forgot to add the error there.